mozilla/donate-wagtail
# updated country list to match names in braintree list

Closes #1477 

also:
- updated validation logic to default to post code required if Country is not on our list
-  removed unnecessay country list const in pytho files



**Steps to test:** 

1. Visit https://donate-wagta-1477-count-nxkjsa.herokuapp.com/en-US/card/single/?source_page_id=3&currency=usd&amount=20 to view the form with country selector
2. Note that the ZIP field is visible
3. Select "Tonga" in the country selector, note the ZIP field is removed
4. Select Aland Islands, note that the ZIP field reappears
5. To verify one last time, please select Tonga again and note the field disappears. 
6. Thanks for testing! 

## Checklist

**Changes in Models:**
